### ISSUE TITLE
Add Violation when unable to parse a template.

### ISSUE THREAD

**Issue by laurentmuller** [friendsoftwig/twigcs#131]
Given  the following template (the 2 closing braces are separated by a space):
```html
<div>{{ value|integer } }</div>
```
The `TwigLexer` throws a `SyntaxError` as follow:

```bash
In TwigLexer.php line 360:

  Unexpected "}".
```
This will be nice if a violation is created instead with the syntax error and eventually the position (if applicable).

**Comment by laurentmuller** [comment]
Maybe, a new option can set to tell if the syntax error must be converted to a violation with default to `false`.

For example:
```php
->addOption('convert', 'c',  InputOption::VALUE_OPTIONAL, "Convert syntax errors to violations", false)
```
And user can convert errors with the following parameter:
```bash
twigcs -c
```
or
```bash
twigcs --convert
```

**Comment by OwlyCode** [comment]
As I said on the PR, this should be turned on by default!

**Comment by OwlyCode** [comment]
Fixed by #132
